static/js/util/util_test.js

```diff
@@ -538,4 +539,30 @@ describe('utility functions', () => {
       assert.equal(classify(undefined), '');
     });
   });
+
+  describe('getUserDisplayName', () => {
+    let profile;
+    beforeEach(() => {
+      profile = {
+        username: 'jane_username',
+        first_name: 'jane',
+        last_name: 'doe',
+        preferred_name: 'test'
+      };
+    });
+
+    it('assert first and last name', () => {
+      assert.equal('jane doe (test)', getUserDisplayName(profile));
+    });
+
+    it('returns username when first not available', () => {
+      profile.first_name = null;
+      assert.equal('jane_username doe (test)', getUserDisplayName(profile));
+    });
+
+    it('not return preferred_name when preferred_name not available', () => {
```

Comment:
please change this `it` text to: `does not show preferred_name when that value is blank`